pkg/services/object/get/get.go

```diff
@@ -110,6 +112,9 @@ func (exec *execCtx) analyzeStatus(execCnr bool) {
 	case statusVIRTUAL:
 		exec.log.Debug("requested object is virtual")
 		exec.assemble()
+		if errors.Is(exec.err, apistatus.ErrObjectNotFound) {
+			exec.analyzeStatus(execCnr)
```

Comment:
This can lead to infinite recursion, suppose you can't get some object's part -- you'd get the same "not found" error and will retry indefinitely.